migrations/0018_create_rsvps.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS rsvps (
+    id INTEGER PRIMARY KEY NOT NULL,
+    user_id INTEGER NOT NULL,
+    event_id INTEGER NOT NULL,
+    ticket_id INTEGER NOT NULL,
+    transaction_id INTEGER,
+    price INTEGER,
```

Comment:
normalize with transactions.price?

Reply:
figured it would be a bit more convenient to avoid the join and having to define a new struct for that, vs. just `Rsvp::list()` and get everything relevant (just this one extra field)